src/osgText/TextBase.cpp

```diff
@@ -424,16 +424,18 @@ osg::BoundingBox TextBase::computeBoundingBox() const
 
     if (_textBBWithMargin.valid())
     {
-        bbox.expandBy(_textBBWithMargin.corner(0)*_matrix);
-        bbox.expandBy(_textBBWithMargin.corner(1)*_matrix);
-        bbox.expandBy(_textBBWithMargin.corner(2)*_matrix);
-        bbox.expandBy(_textBBWithMargin.corner(3)*_matrix);
-        bbox.expandBy(_textBBWithMargin.corner(4)*_matrix);
-        bbox.expandBy(_textBBWithMargin.corner(5)*_matrix);
-        bbox.expandBy(_textBBWithMargin.corner(6)*_matrix);
-        bbox.expandBy(_textBBWithMargin.corner(7)*_matrix);
+        if (!_matrix.isIdentity())
```

Comment:
_matrix being identity can be a valid setting for it so this PR will introduce a bug.